src/main/twirl/views/renderers/manage/guidancePM.scala.html

```diff
@@ -0,0 +1,45 @@
+@(opportunityId:OpportunityId)
+
+
+@main(s"Guidance on grants - RIFS") {
+<div id="middleframe">
+
+    <div class="gidrow">
+        <div class="column-two-thirds">
```

Comment:
We don't need the grid layout for this page, as it doesn't have the RHS column.   You should be able to lose the following divs: middle frame, gidrow, column-two-thirds.

You should add a "text" class to the container enclosing the content to wrap the text at about 8 words in length.